frontend/src/component/feature/StrategyTypes/NewStrategyVariants.tsx

```diff
@@ -61,18 +69,6 @@ export const NewStrategyVariants: FC<{
         };
     }, [JSON.stringify(variantsEdit)]);
 
-    useEffect(() => {
```

Comment:
move this to useState init value. It was running before we set the local variants and resetting everything to 0 variants